gobench.go

```diff
@@ -69,8 +69,7 @@ func doPostDocs(col driver.Collection) {
 	times := make([]time.Duration, nrRequests, nrRequests)
 	wg := sync.WaitGroup{}
 
-	worker := func(times []time.Duration, nrRequestsPerWorker int) {
-		defer wg.Done()
+	worker := func(times []time.Duration) {
```

Comment:
Will `times[i]` in this for loop go out of bounds?
There is no hard relation between `nrRequestsPerWorker` and `len(times)`

Reply:
I believe there is: I ensure that `nrRequests` is the same as `nrRequestsPerWorker * parallelism`.
Then I create (the outer) `times` with length `nrRequests` and I hand in to the go routines slices which are disjoint but together cover the whole of `times`. Unfortunately, I call this `times` as well in the `worker`, will fix that.